service/src/main/java/uk/gov/hmcts/reform/fpl/service/orders/ManageOrdersEventBuilder.java

```diff
@@ -16,17 +22,30 @@
 @RequiredArgsConstructor(onConstructor_ = {@Autowired})
 public class ManageOrdersEventBuilder {
     private final SealedOrderHistoryService historyService;
+    private final List<AmendedOrderFinder<? extends AmendableOrder>> finders;
 
-    public Optional<GeneratedOrderEvent> build(CaseData caseData, CaseData caseDataBefore) {
+    public Optional<ManageOrdersEvent> build(CaseData caseData, CaseData caseDataBefore) {
```

Comment:
I would change this from returning optional to just returning `ManageOrdersEvent`, we are always going to want an event returned so if there isn't one then it is unexpected and we should probs throw an exception